lms/envs/common.py

```diff
@@ -605,6 +605,17 @@
     # toggle does not have a target removal date.
     'ENABLE_LOGISTRATION_MICROFRONTEND': False,
 
+    # .. toggle_name: ENABLE_LEARNER_RECORDS
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: True
+    # .. toggle_description: Enable learner records for the whole platform. This setting maybe overridden by site- and
+    #   org-specific site configurations with the same name.
+    # .. toggle_warnings: Enabling this feature requires that the definition of the ``CREDENTIALS_PUBLIC_SERVICE_URL``
+    #   setting.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2020-10-01
+    'ENABLE_LEARNER_RECORDS': True,
```

Comment:
Quick comment is that this should use ToggleSetting (or Dict) and a default without these common changes. It should live in the most appropriate app. 

I may not get to a deeper review until next week.

Reply:
Indeed! At the time this PR was created the SettingDictToggle did not exist ;)